src/engraving/layout/layoutsystem.cpp

```diff
@@ -159,6 +187,19 @@ System* LayoutSystem::collectSystem(const LayoutOptions& options, LayoutContext&
                 system->removeLastMeasure();
                 ctx.curMeasure->setParent(oldSystem);
             }
+            // If the last appended measure caused a re-layout of the previous measures, now that we are
+            // removing it we need to re-layout the previous measures again. 
```

Comment:
the trailing space it causeing the code style check to fail (yes, it is pretty picky)